docs/references/javascript/sign-up/verification.mdx

```diff
@@ -22,7 +22,7 @@ The `prepareVerification` is used to initiate the verification process for a fie
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `strategy` | `'phone_code' \| 'email_code' \| 'email_link' \| 'saml' \| 'oauth_<provider>' \| 'web3_metamask_signature'` | The verification strategy to validate the user's sign-up request.<br/>The following strategies are supported:<br/><ul><li>`phone_code`: Send an SMS with a unique token to input.</li><li>`email_code`: Send an email with a unique token to input.</li><li>`email_link`: Send an email with a link which validates sign-up</li><li>`saml`: Authenticate against SAML. Experimental</li><li>`oauth_<provider>`: Authenticate against various [OAuth providers](/docs/references/javascript/types/oauth-strategy).</li><li>`web3_metamask_signature`: The verification will attempt to be completed using the user's web3 wallet public address. The `web3_wallet_id` parameter can also be specified to select which of the user's known web3 wallets will be used. Currently Clerk supports [Metamask](https://metamask.io/).</li></ul> |
+| `strategy` | `'phone_code' \| 'email_code' \| 'email_link' \| 'saml' \| 'oauth_<provider>' \| 'web3_metamask_signature'` | The verification strategy to validate the user's sign-up request.<br/>The following strategies are supported:<br/><ul><li>`phone_code`: Send an SMS with a unique token to input.</li><li>`email_code`: Send an email with a unique token to input.</li><li>`email_link`: Send an email with a link which validates sign-up</li><li>`saml`: The user will be authenticated with SAML. **Experimental**</li><li>`oauth_<provider>`: The user will be authenticated with their social sign in account. See available [OAuth providers](/docs/references/javascript/types/oauth#o-auth-provider).</li><li>`web3_metamask_signature`: The verification will attempt to be completed using the user's web3 wallet public address. The `web3_wallet_id` parameter can also be specified to select which of the user's known web3 wallets will be used. Currently Clerk supports [Metamask](https://metamask.io/).</li></ul> |

```

Comment:
I just noticed the hash is `#o-auth-provider`. That's really weird that it got broken out like that, but it works, so 👍